service/src/main/resources/db/migration/R__koski_views.sql

```diff
@@ -32,6 +32,8 @@ TABLE (
         AND placement.type IN ('PRESCHOOL', 'PRESCHOOL_DAYCARE', 'PRESCHOOL_CLUB', 'PREPARATORY', 'PREPARATORY_DAYCARE')
         WINDOW child AS (PARTITION BY child_id)
     ) p
+    JOIN person pr ON p.child_id = pr.id
```

Comment:
Is this join necessary? We're joining based on `pr.id` and then later only using `pr.id` which is the same as `p.child_id` without any joins.

Reply:
Removed join